packages/steiger-plugin-fsd/src/index.ts

```diff
@@ -31,3 +34,19 @@ export default [
   sharedLibGrouping,
   noProcesses,
 ]
+
+const allRulesEnabledConfig: ConfigObject = {
+  rules: allRules.reduce((acc, rule) => ({ ...acc, [rule.name]: 'error' }), {}),
+}
+
+export const plugin: Plugin = {
+  meta: {
+    name: 'steiger-plugin-fsd',
+    version: packageJson.version,
+  },
+  ruleDefinitions: allRules,
+}
+
+export const configs: Record<string, Config> = {
```

Comment:
issue: these are named exports, and I suspect that will promote a different way of usage:
```ts
import { configs } from '@feature-sliced/steiger-plugin'

// use `configs.recommended`
```

I think this is slightly problematic because if you have two plugins, you have to do awkward renames.

suggestion: make one default export that consumers can call whatever they want. I think this code reads better:
```ts
import fsd from '@feature-sliced/steiger-plugin'

// use `fsd.configs.recommended`
```

Reply:
Yes, the current shape of exports does not correspond to that set out in the proposal document. That shape makes perfect sense. I'll make the needed changes. Thanks for catching it!